### ISSUE TITLE
Add tool to compress data to the formats supported by the BIOS

### ISSUE THREAD

**Issue by AntonioND** [blocksds/sdk#149]
We could simply take the compressors by CUE: https://gbatemp.net/download/nintendo-ds-gba-compressors-by-cue.37133/

They are licensed under the GPLv3, so they are FOSS, and they can easily be integrated with the other tools of this SDK.

**Comment by AntonioND** [comment]
Asie has pointed out to me that there is "wf-nnpack", https://github.com/WonderfulToolchain/wonderful-packages/blob/main/packages/wf-nnpack/PKGBUILD We can probably just use them right away. Or, at least, use them as base.

In any case, we also need examples of how to use them.

**Comment by asiekierka** [comment]
Let me know if you do end up using the bundled builds, so I can make `thirdparty-blocksds` have a dependency.

It's easy to build (if you use my forked repository, which makes the software build properly on non-Windows systems).

**Comment by AntonioND** [comment]
Over the last few days I've started to develop an example to use the compressors that come with https://github.com/WonderfulToolchain/wf-nnpack and I've pushed a couple of PRs to the upstream repository https://github.com/PeterLemon/Nintendo_DS_Compressors to improve them. For example, they now take an input and output path rather than replacing the input file by the output data.

As soon as the last PR is merged and `wf-nnpack` is upgraded (and BlocksDS is made to depend on `wf-nnpack`) this issue can be closed.

I've already pushed an example to this repository because I'll be away for a few days, that way everything will be ready when `wf-nnpack` can be updated. https://github.com/blocksds/sdk/commit/528b2639e1f8e787c3422ed266becd29dec96348
